packages/device_info/lib/device_info.dart

```diff
@@ -222,6 +228,9 @@ class IosDeviceInfo {
   /// Unique UUID value identifying the current device.
   final String identifierForVendor;
 
+  ///false, if device is being simulated/emulated
```

Comment:
Nits:

- space after `///`
- "False" or `false` (i.e. wrap in backticks if referring to a Dart keyword).

Reply:
Style:

```dart
/// `false` if the application is running in a simulator, `true` otherwise.
```